config/Make.project.rules

```diff
@@ -190,10 +190,6 @@ define make-flex
 $1/%.cpp: $1/%.l
 	@$(RM) $$@
 	flex $2 -o $$@ $$<
-	@$(MV) $$@ $$@.tail
-	@echo '#include <IceUtil/ScannerConfig.h>' >> $$@
-	@cat $$@.tail >> $$@
-	@$(RM) $$@.tail
```

Comment:
We used this logic to ensure that '#include <...>' was written before Flex included it's own headers.
This is replaced by  %top{ ... } in each of the scanners, which places enclosed code at the tops of files:

https://github.com/InsertCreativityHere/ice/blob/e38e19b7a04bb0f7ebc3e7a60a6983716b986edd/cpp/src/Slice/Scanner.l#L1-L8